sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -22,37 +22,42 @@
 
 == Notice
 
-IMPORTANT: This specification is a draft.
+[%hardbreaks]
+Copyright (C) 2022-2022 Intel Corporation.  All rights reserved.
 
-Copyright (c) 2021-2022 Intel Corporation. All rights reserved.
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
 
-NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are
-trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc.
-used by permission by Khronos.
 
 == Dependencies
 
 This extension is written against the SYCL 2020 specification, Revision 4.
 
 == Status
 
-Draft
-
-This is a preview extension specification, intended to provide early access to
-a feature for review and community feedback. When the feature matures, this
-specification may be released as a formal extension.
-
-Because the interfaces defined by this specification are not final and are
-subject to change they are not intended to be used by shipping software
-products.
+This extension is implemented and fully supported by DPC++.
+[NOTE]
+====
+This extension is currently implemented in {dpcpp} only for GPU devices that support bfloat16 natively. Attempting to use this extension in
+kernels that run on other devices may result in undefined behavior.
+Be aware that the compiler is not able to issue a diagnostic to warn you if this happens.
+====
 
 == Version
 
 Revision: 5
 
-== Introduction
+== Overview
 
-This extension adds functionality to convert value of single-precision
+This extension adds functionality to convert values of single-precision
 floating-point type(`float`) to `bfloat16` type and vice versa. The extension
 doesn't add support for `bfloat16` type as such, instead it uses 16-bit integer
 type(`uint16_t`) as a storage for `bfloat16` values.
```

Comment:
We should also say something about the format of the `bfloat16` type.  A reader of this spec may wonder how `bfloat16` differs from `sycl::half` since they are both 16-bit fp types.

Reply:
Done.